runtimes/web/src/index.js

```diff
@@ -473,7 +473,9 @@ async function loadCartWasm () {
 
         if (deltaFrame >= INTERVAL) {
             lastFrame = now - (deltaFrame % INTERVAL);
-            runtime.update();
+            if (!runtime.crashed) {
+                runtime.update();
+            }
```

Comment:
This doesn't actually STOP the requestAnimation loop, but I think that's ok since it essentially turns it into just a NOP.